libafl/src/inputs/encoded.rs

```diff
@@ -15,7 +15,7 @@ use core::{
 
 use ahash::RandomState;
 use hashbrown::HashMap;
-#[cfg(feature = "std")]
+#[cfg(feature = "regex")]
```

Comment:
I think this is the only place that needs the change from `std` to `regex`

Reply:
sure then looks good after that change

Reply:
Reverted!